### PR TITLE
Split a polyline with a certain number of segment.

### LINKED ISSUE #952: modeler insert segmented Spline
tried the following: 

```python
from pyaedt.desktop import Desktop
from pyaedt import Q3d
from pyaedt.modeler.Primitives import PolylineSegment

d = Desktop("2022.1", False, False)
q3d = Q3d(designname="test", specified_version="2022.1")
q3d.modeler.model_units = "mm"
q3d.modeler.delete()

point = [[1., 1., 0], [1., 2., 1.], [1., 3., 1.]]

line1 = q3d.modeler.create_polyline([[0, 0, 0], point[0]])

line1.insert_segment(position_list=point,
                     segment=PolylineSegment("Spline", num_points=3, num_seg=3))

```
And insert part is not segmented.
image

Need to insert segmented Spline.

in PolylineSegment spine has no num_seg
image

### LINKED ISSUE #975: CreatePolyline does not work if segment is added before origin point
If the first point provided to the parameter `position_list` in the `insert_segment()` method is a a list of integers such as `[0,0,0]`, you can't insert the segment before the existing line.
It is because the list is used to determine if the segment must be added before or after the existing line. It is compared to the vertex position which return a list of floats such as `[0.,0.,0.]`.
So the two lists will never be equal because they are not handling the same type.

### PR DESCRIPTION
**Pull request by MaxJPRey**
Add the ability to split a polyline with a certain number of segment.
Fix #952 .
Fix #975 .

### CODE REVIEW

**Review by maxcapodi78 — CHANGES_REQUESTED** [review]
please add assert in unittest

File: _unittest/test_02_3D_modeler.py [hunk 1]
```diff
@@ -486,3 +487,23 @@ def test_48_coordinate_systems_parametric(self):
         cs4 = self.aedtapp.modeler.create_coordinate_system(name="CSP4", mode="zxz", phi=43, theta="126deg", psi=0)
         tol = 1e-9
         assert sum([abs(x1 - x2) for (x1, x2) in zip(cs3.quaternion, cs4.quaternion)]) < tol
+
```
[the remark this answers is not in the document]
Comment (MaxJPRey) [reply _unittest/test_02_3D_modeler.py:490]: Sure I will do.

[the remark this answers is not in the document]
Comment (MaxJPRey) [reply _unittest/test_02_3D_modeler.py:490]: @maxcapodi78 Done. Thanks for the suggestion.

**Review by maxcapodi78 — APPROVED** [review]
LGTM

File: _unittest/test_02_3D_modeler.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (MaxJPRey) [reply _unittest/test_02_3D_modeler.py:490]: @maxcapodi78 I used the `get_oo_property_value()` in the test assertion to make sure that the modeler operations were correct.

File: _unittest/conftest.py [hunk 2]
```diff
@@ -70,7 +70,7 @@
         default_version = sys.modules["__main__"].oDesktop.GetVersion()[0:6]
     config = {
         "desktopVersion": default_version,
-        "NonGraphical": True,
+        "NonGraphical": False,
```
Comment (maxcapodi78) [inline _unittest/conftest.py:73]:
```suggestion
        "NonGraphical": True,
```

**Review by maxcapodi78 — CHANGES_REQUESTED** [review]
Unit Tests has to run in nongraphical

File: _unittest/conftest.py (hunk 2 above)
Comment (MaxJPRey) [reply _unittest/conftest.py]: Thanks @maxcapodi78 . It was modified for debugging purpose as I was working with the modeler and wanted to make sure that the spline were properly drawn.

**Review by maxcapodi78 — APPROVED** [review]

**Pull request merged**